ugs-platform/ugs-platform-plugin-jog/src/main/java/com/willwinder/ugs/nbp/jog/JogTopComponent.java

```diff
@@ -53,8 +53,8 @@ This file is part of Universal Gcode Sender (UGS).
         preferredID = "JogTopComponent"
```

Comment:
This is missing the `persistenceType = TopComponent.PERSISTENCE_ALWAYS ` property, I think setting that helps the UI load faster.

Reply:
This is the default value so I didn't bother setting it. Would you like to have it set anyway?

TopComponent:
```java
public static @interface Description {
        /** Default value for {@link TopComponent#getPersistenceType()}. 
         * 
         * @return one of {@link TopComponent#PERSISTENCE_ALWAYS}, 
         * {@link TopComponent#PERSISTENCE_NEVER}, 
         * {@link TopComponent#PERSISTENCE_ONLY_OPENED}
         * 
         */
        public int persistenceType() default PERSISTENCE_ALWAYS;
    }
```